geomesa-hbase/geomesa-hbase-tools/src/main/scala/org/locationtech/geomesa/hbase/tools/ingest/HBaseBulkIngestCommand.scala

```diff
@@ -16,22 +16,25 @@ import org.locationtech.geomesa.hbase.data.HBaseDataStore
 import org.locationtech.geomesa.hbase.jobs.HBaseIndexFileMapper
 import org.locationtech.geomesa.hbase.tools.ingest.HBaseBulkIngestCommand.HBaseBulkIngestParams
 import org.locationtech.geomesa.hbase.tools.ingest.HBaseIngestCommand.HBaseIngestParams
+import org.locationtech.geomesa.tools.DistributedRunParam.RunModes
 import org.locationtech.geomesa.tools.ingest.AbstractIngest.StatusCallback
-import org.locationtech.geomesa.tools.ingest.{ConverterIngest, ConverterIngestJob}
+import org.locationtech.geomesa.tools.ingest.{ConverterCombineIngestJob, ConverterIngest, ConverterIngestJob}
 import org.locationtech.geomesa.tools.{Command, OutputPathParam, RequiredIndexParam}
 import org.locationtech.geomesa.utils.index.IndexMode
 import org.opengis.feature.simple.SimpleFeatureType
+import java.util.{List => jList}
+
 
 class HBaseBulkIngestCommand extends HBaseIngestCommand {
 
   override val name = "bulk-ingest"
   override val params = new HBaseBulkIngestParams()
 
-  override protected def createConverterIngest(sft: SimpleFeatureType, converterConfig: Config): Runnable = {
+  override protected def createConverterIngest(sft: SimpleFeatureType, converterConfig: Config, ingestFiles: jList[String]): Runnable = {
```

Comment:
can we make the method take a Seq instead of a java collection?

Reply:
Yeah, I kept it as a jList because that's what the input files param was and this would alter less code.